### ISSUE TITLE
Question on AWS Lambda Plugin and Proxy mode

### ISSUE THREAD

**Issue by olconstant** [Kong/kong#2713]
### Summary

Hi,

In my company, we are implementing Kong with the AWS Lambda Plugin.
But we written our lambda handlers to process same inputs as AWS API Gateway in Proxy mode:
```
{
  "httpMethod": "POST",
  "path": "/",
  "body": "{\"test\":\"body\"}",
  "queryStringParameters": {
    "foo": "bar"
  },
  "headers": {
    "Content-Type": "application/json"
  }
}
```

I am wondering if Kong AWS Lambda Plugin can support RequestResponse in Proxy mode as well?
It will allow to forward the query string to the lambda function, and to manage routing and middleware inside the handler.

I tried to use Kong Request Transformer Plugin, but it doesn't allow to dynamically lookup the request values.

I found a question related to this topic, but it's already closed: https://github.com/Mashape/kong/issues/2329

What do you think? Any other approach to suggest?

**Comment by Mgutjahr** [comment]
Hi @olconstant, 
definitely a +1 for proxy mode! 
Did you have a look at the pull request #2470? This change does not directly implement the proxy mode but it allows you to forward the request uri, request method and headers to the lambda function. Depending on what you need this could be already enough.

**Comment by thibaultcha** [comment]
Hi,

Thanks @Mgutjahr for pointing our this PR! Indeed, this has been released in our most recent release, 0.11.1.

I believe this will answer your question/requirements @olconstant? As such, I will go ahead and close this issue, but feel free to contact our other community support channels, like Gitter or the mailing list (see README.md) for troubleshooting questions - we wish to reserve our GitHub issues for bug reports only. Thank you!
